### PR TITLE
PXB-3132 Improve documentation of the option compress-zstd-level (8.0)

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
modified:   docs/create-compressed-backup.md
	modified:   docs/xtrabackup-option-reference.md

### CODE REVIEW

File: docs/xtrabackup-option-reference.md [hunk 1]
```diff
@@ -171,7 +179,10 @@ that will read the data and pipe it to 2 compression threads.
 
 ### --compress-zstd-level(=#)
 
-This option specifies `ZSTD` compression level. The default value is 1. Allowed range of values is from 1 to 19. 
+This option specifies `ZSTD` compression level. Compression levels provide a trade-off between the compression speed and the compressed files' size. A lower compression level provides faster compression speed but larger file sizes. A higher compression level provides lower compression speed but smaller file sizes. For example, set level 1 if the compression speed is the most important for you. Set level 19 if the compressed file size is the most important.
```
Comment (satya-bodapati) [inline docs/xtrabackup-option-reference.md]: remove the single quote after the files. You can write it as compressed file size.

**Review by satya-bodapati — APPROVED** [review]
Please check the typo and fix it.
Ok to merge after fixing it.

File: docs/xtrabackup-option-reference.md (hunk 1 above)
Comment (alina-derkach-oaza) [reply docs/xtrabackup-option-reference.md]: Fixed

File: docs/create-compressed-backup.md [hunk 2]
```diff
@@ -66,7 +66,7 @@ To make a compressed backup, use the `--compress` option along with the `--backu
 
     Using qpress/QuickLZ to compress backups is deprecated and may be removed in future versions. We recommend using either `LZ4` or Zstandard (`ZSTD`) compression algorithms. `ZSTD` compression algorithm is in [tech preview](glossary.md#tech-preview). 
 
-??? note "Before Percona XtraBackup 8.0.31-24"
+??? note "Up to Percona XtraBackup 8.0.31-24"
```
Comment (patrickbirch) [inline docs/create-compressed-backup.md]: Up to? 
Before is clearer.

File: docs/xtrabackup-option-reference.md [hunk 3]
```diff
@@ -144,18 +155,15 @@ This option tells xtrabackup to compress all output data, including the transact
...
 
-    You can extract the contents of the files by using the `--decompress` option. 
-
-* Starting with Percona XtraBackup 8.0.31-24 using qpress/QuickLZ to compress backups is deprecated and may be removed in future versions. We recommend using either `LZ4` or Zstandard (`ZSTD`) compression algorithms. `ZSTD` compression algorithm is in [tech preview](glossary.md#tech-preview). See [Create a compressed backup](create-compressed-backup.md) for more information.
+    You can extract the contents of the files by using the `--decompress` option.
 
-* Percona XtraBackup 8.0.34-29 removes `qpress/QuickLZ` and moves the `ZSTD` compression method to [General Availability](glossary.md#general-availability-ga). With this version `ZSTD` becomes the default compression method for the `--compress` option.
+* Starting with Percona XtraBackup 8.0.31-24 the use of qpress/QuickLZ to compress backups is deprecated and may be removed in future versions. We recommend using either `LZ4` or `ZSTD` compression algorithms. `ZSTD` compression algorithm is in [tech preview](glossary.md#tech-preview).
```
Comment (patrickbirch) [inline docs/xtrabackup-option-reference.md]: From Percona XtraBackup 8.0.31-24,

File: docs/xtrabackup-option-reference.md [hunk 4]
```diff
@@ -133,7 +133,18 @@ This option tells xtrabackup to compress all output data, including the transact
 
 **Version updates**
 
-* Before Percona XtraBackup 8.0.31-24 the `--compress` option uses either the `quicklz`, `lz4`, or `ZSTD` compression algorithm to compress all output data. `quicklz` is chosen by default.
+* With Percona XtraBackup 8.0.34-29, qpress/QuickLZ is no longer supported for compress operations. The `ZSTD` compression algorithm is moved to [General Availability](glossary.md#general-availability-ga). With this version, `ZSTD` becomes the default compression method for the `--compress` option.
```
Comment (patrickbirch) [inline docs/xtrabackup-option-reference.md]: From Percona XtraBackup 8.0.34-29,

File: docs/xtrabackup-option-reference.md (hunk 3 above)
Comment (alina-derkach-oaza) [reply docs/xtrabackup-option-reference.md]: updated

File: docs/xtrabackup-option-reference.md (hunk 4 above)
Comment (alina-derkach-oaza) [reply docs/xtrabackup-option-reference.md]: updated

File: docs/create-compressed-backup.md (hunk 2 above)
Comment (alina-derkach-oaza) [reply docs/create-compressed-backup.md]: updated

**Review by patrickbirch — APPROVED** [review]
Good

**Pull request merged**